tests/test_directions.py

```diff
@@ -72,6 +74,8 @@ def test_direction_car(mock_directions_car):
     response_data = response.json()
     assert response_data["status"] == "success"
     assert len(response_data["data"]["routes"]) == 3
+    assert response_data["data"]["routes"][0]["start_time"] == "2018-06-14T08:30:00"
+    assert response_data["data"]["routes"][0]["end_time"] == "2018-06-14T09:00:19.200000"
```

Comment:
I would suggest to use `dt.isoformat(timespec="seconds")` to return a datetime with fixed precision.